packages/nextjs/components/ActivitiesItem.tsx

```diff
@@ -0,0 +1,225 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { ethers } from "ethers";
+import { isAddress } from "ethers/lib/utils";
+import Blockies from "react-blockies";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { hardhat } from "wagmi/chains";
+import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
+import { getBlockExplorerAddressLink, getTargetNetwork } from "~~/utils/scaffold-eth";
+
+/**
+ * Displays an address (or ENS) with a Blockie image and option to copy address.
+ */
+
+const Address = ({ address }: { address: string }) => {
```

Comment:
Why not to use `Address` from `components/scaffold-eth/`?
If there is a reason, please change the name of this component, `Address` already exists

Reply:
Because of the Ui https://www.figma.com/file/D0YZXzXhNtaP8j0lcq2JZR/Dice?type=design&node-id=1%3A2&mode=design&t=9nWbwZgT95vcsicq-1